docs/blog/2020-05-22-happy-fifth-bday-gatsby/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: It’s Gatsby’s 5th Birthday 🎂 (and everyone’s invited!)
+date: 2020-05-22
+author: Kyle Mathews
+excerpt: “Five years ago today, a nascent Gatsby emerged in the form of Issue 1 on the brand new Gatsbyjs GitHub repo. Here are some thoughts about where we began and how Gatsby has evolved during this most interesting demi-decade.”
+tags:
+  - community
+  - gatsby-cloud
+  - plugins
+---
+
+The web is an incredible place. I’m so happy I get to help build it. I’ve been building websites and web apps for a long time now, and I spent a lot of that time thinking about and experimenting with what a perfect toolset for building for the web would look like. Five years ago those thoughts coalesced into the unveiling of a nascent framework I had decided to call Gatsby:
+
+> There's a lot of static site generators out there and I've played with several and written my own for my blog. They're all pretty much the same and not particularly interesting. I think a React.js based SSG can push the state of the art in three ways — easy no-page transitions, react.js style components, and leveraging the growing react.js ecosystem of tools and components.
+> Most stuff on the web are sites, not apps. And react.js components are just as powerful for sites as they are for apps so a kickass tool for building react.js sites would be very valuable.
+
+-- Opened as Issue #1, “Braindump of ideas,” by @KyleAMathews on the brand new Gatsbyjs/Gatsby GitHub repo, May 22, 2015.
+
+Now it’s May 22, 2020. Five years (and 9,820 closed issues) later, it is interesting to pause for a beat to look back at where we started. Where is Gatsby now? What has actually come into existence, from that original brain dump vision, and what fell away?
+
+## Project name: Codex
+
+One of the original tenets of the Gatsby codex (“Codex” was actually my code name for the project when I first started pulling it together) came from Facebook engineering's "pit of success" slogan: Incredibly fast websites should be the default state, not some monumental engineering feat. So one of the foundational philosophies behind building Gatsby was, Anything that prevents Gatsby from generating the fastest possible website is a bug. That understanding guided a lot of decisions along the way, like incorporating GraphQL (thanks again, Facebook!) to allow client code to specify data requirements. So that every page can specify exactly the data it needs so that -- and only that -- data gets loaded with each page. That was a key moment.
```

Comment:
```suggestion
One of the original tenets of the Gatsby codex (“Codex” was actually my code name for the project when I first started pulling it together) came from Facebook engineering's "pit of success" slogan: Incredibly fast websites should be the default state, not some monumental engineering feat. So one of the foundational philosophies behind building Gatsby was, anything that prevents Gatsby from generating the fastest possible website is a bug. That understanding guided a lot of decisions along the way, like incorporating GraphQL (thanks again, Facebook!) to allow client code to specify data requirements. So that every page can specify exactly the data it needs so that -- and only that -- data gets loaded with each page. That was a key moment.
```

Reply:
downcased the A that is in mid-sentence @mgienow